packages/hoppscotch-backend/src/utils.ts

```diff
@@ -173,6 +173,32 @@ export const validateSMTPUrl = (url: string) => {
   return false;
 };
 
+/**
+ * Checks to see if the URL is valid or not
+ * @param url The URL to validate
+ * @returns boolean
+ */
+export const validateUrl = (url: string) => {
+  const urlRegex = /^(http|https):\/\/[^ "]+$/;
+  return urlRegex.test(url);
+};
+
+/**
+ * Validate SSO (Google, Github, Microsoft) Scope
+ * @param scope The scope to validate
+ * @returns boolean
+ */
+export const validateSSOScope = (scope: string) => {
+  if (!scope || scope.length === 0) return false;
+
+  const scopes = scope.split(',');
+  scopes.forEach((aScope) => {
+    if (aScope.length === 0) return false;
+  });
+
+  return true;
+};
+
```

Comment:
I think we need to remove this function, every SSO provider defines scopes in their own way, so even if the ones we use right now can use this function there is no guarantee any new SSO provider we add in the future can use this too.

Reply:
Okay @balub